auction-server/src/opportunity/api.rs

```diff
@@ -211,7 +211,7 @@ pub async fn post_quote(
     State(store): State<Arc<StoreNew>>,
     Json(params): Json<QuoteCreate>,
 ) -> Result<Json<Quote>, RestError> {
-    if get_program(&auth)? != ProgramSvm::Phantom {
+    if get_program(&auth)? != ProgramSvm::Swap {
```

Comment:
I'm not sure about changing this part. We wanna create a different api_key for each of our routers. One for each client.